### PR TITLE
Use schema aliases as a lookup option when importing hashes

### PR DESCRIPTION
**Pull request by arlandism**
Addresses issue #44

### CODE REVIEW

**Comment by iande** [comment]
If I remember right, this will actually simplify something else in the code base.

File: lib/erlen/schema/base.rb [hunk 1]
```diff
@@ -60,8 +60,8 @@ def import(obj)
           obj_attribute_name = (attr.options[:alias] || attr.name).to_sym
 
           if obj.is_a? Hash
-            if obj.has_key?(k) || attr.options.has_key?(:default)
-              attr_val = obj.fetch(k, attr.options[:default])
+            if obj.has_key?(k) || obj.has_key?(obj_attribute_name) || attr.options.has_key?(:default)
```
Comment (timpbrenner) [inline lib/erlen/schema/base.rb]: This feels like an if..else if...else if..else that is masquerading as a single if. I think that would be clearer.

Comment (arlandism) [reply lib/erlen/schema/base.rb]: You cleaned this up a little bit in your PR so gonna hold off until that gets merged

Comment (arlandism) [reply lib/erlen/schema/base.rb]: Done

**Pull request merged**